projects/auth0-angular/src/lib/auth.config.ts

```diff
@@ -1,6 +1,18 @@
 import { CacheLocation, GetTokenSilentlyOptions } from '@auth0/auth0-spa-js';
 import { InjectionToken } from '@angular/core';
 
+/**
+ * Defines a common set of HTTP methods.
+ */
+export const enum HttpMethod {
+  Get = 'GET',
```

Comment:
I'm not 110% convinced about the casing.

The examples I found are either
1. method names using lower. e.g.: `httpClient.post("your URL here")`
2. param values using uppercase. e.g.: `httpClient.request("POST", "your URL here")`

However, this [SO post](https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums) talks about different conventions, so I ended up more confused than I was 😂

Reply:
I think you're confusing the method names of `HttpClient` vs. HTTP verbs.

The methods on `HttpClient` (`get`, `post`, `delete`, etc) follow normal JS camelCase convention. The HTTP verbs/methods themselves (`GET`, `POST`, `DELETE`) appear to be uppercase as we previously discussed and are case sensitive.

There's a third convention at play in this snippet, which is TypeScript enums. These appear in TitleCase in [the TypeScript docs](https://www.typescriptlang.org/docs/handbook/enums.html), so I've gone with that convention here.

Reply:
Sounds good. Thanks for explaining that.